app/2.0/start/first-element/step-3.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Step 3: Use data binding and properties"
+subtitle: "Build your first Polymer element"
+---
+
+Right now, the element doesn't do much. In this step, you'll give it a basic
+API, allowing you to configure the icon from markup, using an attribute, or
+from JavaScript, using a property.
+
+First, a bit of data binding. Find the `<iron-icon>` element and change the value of the `icon` attribute from `"polymer"` to  "`[[toggleIcon]]`".
+
+icon-toggle.html { .caption }
+
+```html
+<!-- local DOM goes here -->
+<iron-icon icon="[[toggleIcon]]">
+</iron-icon>
+```
+
+Key information:
+
+  * `toggleIcon` is a <em>property</em> you'll define on the toggle button element. It doesn't have a default value yet.
+    
+  * The `icon="[[toggleIcon]]" `assignment is a <em>data binding</em>. It links your element's `toggleIcon` <em>property</em> with the `<iron-icon>`'s `icon` property.
+
+You could now use your element and set the `toggleIcon` property in markup or
+using JavaScript. If you're curious about where the new icons come from, take a look at `demo-element.html` in the `demo` folder.
+
+You'll see lines like this:
+
+demo-element.html—existing demo code { .caption }
+
+```html
+<icon-toggle toggle-icon="star" pressed></icon-toggle>
+```
+
+These lines are _already in place_, you don't have to add them. These lines
+are part of the demo you see on screen. But if you'd like to experiment, try
+adding a new `<icon-toggle>` element to the `icon-toggle-demo.html` file. Some
+icon names you can try are `add`, `menu`, and `settings`.
+
+**Learn more: attribute and property names.** You'll note that the markup above
+uses `toggle-icon`, not `toggleIcon`. Polymer represents camelCase property names
+using dash-case attribute names. To learn more, see <a href="/2.0/docs/devguide/properties#property-name-mapping">Property
+name to attribute name mapping</a> in the Polymer library docs.
+{ .alert .alert-info }
+
+Next, add a declaration for the `toggleIcon` property.
```

Comment:
Note: in 2.0, this step is going to need to move up. 2.0 requires the property to be declared in the `properties` object if you want to be able to configure it from markup.